ui/pages/keychains/reveal-seed.js

```diff
@@ -143,7 +144,9 @@ const RevealSeedPage = () => {
           error={error}
           width={BLOCK_SIZES.FULL}
         />
-        {error && <HelpText severity={SEVERITIES.DANGER}>{error}</HelpText>}
```

Comment:
We also can include the migration in `ui/components/component-library/form-text-field/form-text-field.js` since there are not many components using this instance